{
  "abstract": [
    {
      "text": "The name of an exception raised when there is already a print operation in process.",
      "type": "text"
    }
  ],
  "documentVersion": 0,
  "hierarchy": {
    "paths": [
      [
        "doc://com.apple.documentation/documentation/technologies",
        "doc://com.apple.documentation/documentation/foundation",
        "doc://com.apple.documentation/documentation/foundation/errors_and_exceptions",
        "doc://com.apple.documentation/documentation/foundation/nsexception",
        "doc://com.apple.documentation/documentation/foundation/nsexceptionname"
      ]
    ]
  },
  "identifier": {
    "interfaceLanguage": "swift",
    "url": "doc://com.apple.documentation/documentation/foundation/nsexceptionname/1531898-printoperationexistsexception"
  },
  "kind": "symbol",
  "legacy_identifier": 1531898,
  "legalNotices": {
    "copyright": "Copyright &copy; 2025 Apple Inc. All rights reserved.",
    "privacyPolicy": "https://www.apple.com/privacy/privacy-policy",
    "termsOfUse": "https://www.apple.com/legal/internet-services/terms/site.html"
  },
  "metadata": {
    "externalID": "c:@NSPrintOperationExistsException",
    "fragments": [
      {
        "kind": "text",
        "text": "static let "
      },
      {
        "kind": "identifier",
        "text": "printOperationExistsException"
      },
      {
        "kind": "text",
        "text": ": NSExceptionName"
      }
    ],
    "modules": [
      {
        "name": "AppKit"
      }
    ],
    "parent": {
      "title": "NSExceptionName"
    },
    "platforms": [
      {
        "current": "15.4",
        "introducedAt": "10.0",
        "name": "macOS"
      }
    ],
    "role": "symbol",
    "roleHeading": "Type Property",
    "symbolKind": "structdata",
    "title": "printOperationExistsException"
  },
  "primaryContentSections": [
    {
      "declarations": [
        {
          "languages": [
            "swift"
          ],
          "platforms": [
            "macOS"
          ],
          "tokens": [
            {
              "kind": "keyword",
              "text": "static"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "keyword",
              "text": "let"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "identifier",
              "text": "printOperationExistsException"
            },
            {
              "kind": "text",
              "text": ": "
            },
            {
              "identifier": "doc://com.apple.documentation/documentation/foundation/nsexceptionname",
              "kind": "typeIdentifier",
              "preciseIdentifier": "c:@T@NSExceptionName",
              "text": "NSExceptionName"
            }
          ]
        }
      ],
      "kind": "declarations"
    },
    {
      "content": [
        {
          "anchor": "discussion",
          "level": 2,
          "text": "Discussion",
          "type": "heading"
        },
        {
          "inlineContent": [
            {
              "text": "The methods that raise this exception are the ",
              "type": "text"
            },
            {
              "code": "EPSOperation...",
              "type": "codeVoice"
            },
            {
              "text": " and ",
              "type": "text"
            },
            {
              "code": "printOperation...",
              "type": "codeVoice"
            },
            {
              "text": ".",
              "type": "text"
            }
          ],
          "type": "paragraph"
        }
      ],
      "kind": "content"
    }
  ],
  "references": {
    "doc://com.apple.documentation/documentation/foundation": {
      "identifier": "doc://com.apple.documentation/documentation/foundation",
      "kind": "symbol",
      "role": "collection",
      "title": "Foundation",
      "type": "topic",
      "url": "/documentation/foundation"
    },
    "doc://com.apple.documentation/documentation/foundation/errors_and_exceptions": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/errors_and_exceptions",
      "kind": "article",
      "role": "collectionGroup",
      "title": "Errors and Exceptions",
      "type": "topic",
      "url": "/documentation/foundation/errors_and_exceptions"
    },
    "doc://com.apple.documentation/documentation/foundation/nsexception": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsexception",
      "kind": "symbol",
      "role": "symbol",
      "title": "NSException",
      "type": "topic",
      "url": "/documentation/foundation/nsexception"
    },
    "doc://com.apple.documentation/documentation/foundation/nsexceptionname": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsexceptionname",
      "kind": "symbol",
      "role": "symbol",
      "title": "NSExceptionName",
      "type": "topic",
      "url": "/documentation/foundation/nsexceptionname"
    },
    "doc://com.apple.documentation/documentation/technologies": {
      "identifier": "doc://com.apple.documentation/documentation/technologies",
      "kind": "technologies",
      "title": "Technologies",
      "type": "topic",
      "url": "/documentation/technologies"
    }
  },
  "schemaVersion": {
    "major": 0,
    "minor": 3,
    "patch": 0
  },
  "sections": [],
  "variants": [
    {
      "paths": [
        "documentation/appkit/nsprintoperationexistsexception"
      ],
      "traits": [
        {
          "interfaceLanguage": "occ"
        }
      ]
    },
    {
      "paths": [
        "documentation/foundation/nsexceptionname/1531898-printoperationexistsexception"
      ],
      "traits": [
        {
          "interfaceLanguage": "swift"
        }
      ]
    }
  ]
}
